superset-frontend/src/components/DvtNavigationBar/index.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import DvtNavigationBarItem from '../DvtNavigationBarItem';
+import {
+  StyledNavigationBar,
+  StyledNavigationBarItem,
+} from './dvt-navigation-bar.module';
+
+export interface DvtNavigationBarProps {
+  data: [
+    {
+      icon: string;
+      label: string;
+      url?: string;
+    },
+  ];
+  active?: string;
+  setActive?: (newUrl: string) => void;
+}
+
+const DvtNavigationBar: React.FC<DvtNavigationBarProps> = ({
+  data,
+  active,
+  setActive,
+}) => (
+  <StyledNavigationBar>
+    {data.map((item, index) => (
+      <StyledNavigationBarItem>
+        <DvtNavigationBarItem
```

Comment:
Expected an assignment or function call and instead saw an expression. sebebi

sen data.map sonra div sonra div key verdiğinde hatası verdi. Data.map sonra ilk her zaman key vermeli.

Reply:
bu önemli değil, StyledNavigationBarItem kaldırıp ve StyledNavigationBar gap verelim. Sen margin-bottom: 32px; verdiğinde gibi

Reply:
DvtNavigationBarItem içinde key={index} kalsın sorunsuz olur